### PR TITLE
minor grammar fixes

### PR DESCRIPTION
**Pull request by PaddyOH**
Question re lines 24 and 25 -- are they necessary? They show up in the introduction apparently out of nowhere. Intro text refers to a setup wizard with Jenkins 2, but info tag identifies a Plugin Install Wizard instead. If Plugin Info and Setup are the same wizard, that's not clear:
        %pre.code
          INFO: This is a new Jenkins instance. The Plugin Install Wizard will be launched.

### CODE REVIEW

**Comment by PaddyOH** [comment]
@reviewbybees

File: content/download.html.haml [hunk 1]
```diff
@@ -29,7 +29,7 @@ title: Jenkins installation and setup
         %p
           Jenkins requires Java7 or above and Servlet 3.1 to function. Java8
           is recommended. Jenkins requires a fair amount of memory to operate
-          well. Smaller installations should start around 256MB-1GB.
+          well. Smaller installations should start around 256 MB-1 GB.
```
Comment (jglick) [inline content/download.html.haml]: n-dash `–`

File: content/download.html.haml [hunk 2]
```diff
@@ -69,11 +69,11 @@ title: Jenkins installation and setup
         %p
           The manual way is to follow Installing Jenkins as a Windows
           service. Alternatively, you can install a servlet container like
-          GlassFish and Tomcat, which can run as a service by itself, and
+          GlassFish or Tomcat, which can run as a service by itself, and
           then deploy Jenkins to it.
 
         %p
-          Since Jenkins was written to work on unix-like platforms, some
+          Because Jenkins was written to work on unix-like platforms, some
```
Comment (jglick) [inline content/download.html.haml]: Unix

Comment (PaddyOH) [inline content/download.html.haml]: fixed

**Pull request merged**